### ISSUE TITLE
Manual testing against air-gapped environment

### ISSUE THREAD

**Issue by rm3l** [janus-idp/operator#130]
This is to make sure we can claim that the operator supports disconnected environments.

Protocol for testing is depicted in https://github.com/janus-idp/operator/issues/80#issuecomment-1862880078_:

---
Testing required:
* install with operator
* install with operator using postgres; verify the postgres pod starts and can be used

Pseudo-code for a test:
* start a cluster
* mirror images in the CSV to that cluster according to https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.10/html-single/administration_guide/index#installing-devspaces-in-a-restricted-environment or https://docs.openshift.com/container-platform/4.13/installing/disconnected_install/installing-mirroring-installation-images.html
* add catalog source for the operator subscription
* "turn off the internet" via whatever proxy or DNS config works for you
* perform operator install
* verify pods all start (including postgres and dev hub)
* verify RHDH loads (route works)

**Comment by gazarenkov** [comment]
@rm3l 
How do we deploy Operator to Openshift Cluster here?
I believe we need to use OperatorHub method (with Catalog etc), do we have a procedure for that?

**Comment by rm3l** [comment]
> @rm3l How do we deploy Operator to Openshift Cluster here? I believe we need to use OperatorHub method (with Catalog etc), do we have a procedure for that?

Whoops - missed your comment. But I think we already discussed this via Slack.

**Comment by rm3l** [comment]
Manual checks summarized in this document: https://docs.google.com/document/d/1BpBQAm8BMb9xgqMPuIKdXpgBT1a1DYj_IsziXYsghGc/edit

TL;DR is that it works as expected thanks to the [`oc adm catalog mirror`](https://github.com/openshift/oc/blob/master/pkg/cli/admin/catalog/mirror.go) ([doc](https://docs.openshift.com/container-platform/4.14/installing/disconnected_install/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images)) command, which can read the related images from the ClusterServiceVersion and mirror them to a target registry. After disconnecting the cluster, the operator and operands can be installed and work fine.

/close
